src/main/java/com/thebois/views/GameScreen.java

```diff
@@ -47,6 +51,8 @@ public void render(float delta) {
         camera.update();
         batch.setProjectionMatrix(camera.combined);
 
+        worldView.draw(batch, rectangleSize);
+
         batch.begin(ShapeRenderer.ShapeType.Filled);
         batch.setColor(blueColor);
         batch.rect(worldWidth / 2 - rectangleSize, worldHeight / 2 - rectangleSize, rectangleSize,
```

Comment:
Good choice to keep it, in case we don't have any finished pawns by Friday.

I suppose whomever implements `Being` rendering will have to remove the rectangle example code?

Reply:
When we have a demo until we can actully play the game. So yes, when a Being is renderd (and walking) we can replace this this demo with that.